node/nodeMemoryConfig_test.go

```diff
@@ -0,0 +1,47 @@
+package node
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/pelletier/go-toml"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMemoryConfig(t *testing.T) {
+	nodeConfig := config.Config{}
+
+	tomlString, err := ioutil.ReadFile("../cmd/node/config/config.toml")
```

Comment:
loading the real config for this unit test... :thinking: 
this pattern can be used as an early warning for a misconfigured config.toml file but will make tests depend on a physical file. Can be left as it is, will think about this